lib/season_stats.rb

```diff
@@ -24,24 +24,24 @@ def games_by_season_id(season_id)
 
   def team_name_by_team_id
     all_teams = {}
-    @teams.each do |team|
-      all_teams[team.team_id] = team.team_name
-    end
+    @teams.each { |team| all_teams[team.team_id] = team.team_name }
     all_teams
   end
 
   def games_by_team_name(season_id)
-    games_by_id = games_by_season_id(season_id).group_by {|game| game.team_id}
+    games_by_id = games_by_season_id(season_id).group_by { |game| game.team_id }
     games_by_id.transform_keys(&team_name_by_team_id.method(:[]))
   end
 
-  def winning_percentage_by_head_coach(season_id)
-    games_by_head_coach = games_by_season_id(season_id).group_by { |game| game.head_coach}
+  def games_by_head_coach(season_id)
+    games_by_season_id(season_id).group_by { |game| game.head_coach }
+  end
 
```

Comment:
I like this helper method, it is easier to follow when this is outside of the winning percentage method.